web_monitoring/pf_edgi.py

```diff
@@ -20,6 +20,17 @@ def list_cabinets():
     assert content['status'] == 'ok'  # business logic is OK
     return content['cabinets']
 
+def get_cabinetID(url):
```

Comment:
Minor quibble: can it be `cabinet_id` for consistency with the existing function signatures (such as `list_archives(cabinet_id)`?